options/locale/locale_en-US.ini

```diff
@@ -1417,7 +1417,7 @@ issues.due_date_remove = "removed the due date %s %s"
 issues.due_date_overdue = "Overdue"
 issues.due_date_invalid = "The due date is invalid or out of range. Please use the format 'yyyy-mm-dd'."
 issues.dependency.title = Dependencies
-issues.dependency.issue_no_dependencies = This issue currently doesn't have any dependencies.
+issues.dependency.issue_no_dependencies = No dependencies set.
 issues.dependency.pr_no_dependencies = This pull request currently doesn't have any dependencies.
```

Comment:
Should we update this translation as well then?

Reply:
Yes, however, I'm not sure how to go about that.

Reply:
With the same style of writing? So that it will be `No dependencies set.`

Reply:
Yes, how do we update the translations? Is there an automated process?

Reply:
Need to change is needed to other languages, only english

Reply:
Yeah, I'd resolve this comment if I could.
However, for an unknown reason, the author of PR comments cannot resolve his own comments on GitHub...